send_email.py

```diff
@@ -0,0 +1,37 @@
+import smtplib, ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+sender_from_email = "sender@gmail.com"
+receiver_to_email = "receiver@gmail.com"
+password = input("Type your password and hit enter:")
+
+message = MIMEMultipart("alternative")
+message["Subject"] = "Network Analysis Report"
+message["From"] = sender_from_email
+message["To"] = receiver_to_email
+
+# Create the HTML version of email content message
```

Comment:
Could you please include the files that are generated when running the network_test_class.py or the Analytics.ipynb. They are located in webpage/figures (but only if analytics has been run)

Reply:
which type of files are generated?? And do we need them as attachments??

Reply:
If you run the analytics after network_test performed for a while, 3 diagrams are generated.

Reply:
I have added a logic to attach all the reports in email. please have a look.